src/WebJobs.Script.WebHost/Controllers/ExtensionsController.cs

```diff
@@ -115,6 +118,14 @@ public async Task<IActionResult> GetJobs()
             return Ok(result);
         }
 
+        [HttpGet]
+        [Route("admin/host/extensions/config")]
```

Comment:
The config isn't restricted to just extensions - e.g. you're also including ConcurrencyOptions. So the route should be admin/host/config

Reply:
Done